man/docker-run.1.md

```diff
@@ -548,6 +553,15 @@ the exit codes follow the `chroot` standard, see below:
 
 # EXAMPLES
 
+## Running container in Read-Only mode
```

Comment:
Here, I think you need to use individual sentences to talk about the reason a user would do this.  Also "probably" and "maybe" aren't the best terms for tech docs.  Something like puts the reason, the solution, and the example in a better order:

```markdown
## Running container in read-only mode

If you want to secure a container and protect its content from modification,
you can use the --read-only switch.  If you protect the container, you should make
allowances so that applications in the container have a space to write temporary
data. The best way to do this is to mount tmpfs directories in /run and also
/tmp.

    # docker run -v --read-only -v tmpfs:/run -v tmpfs:/tmp -i -t fedora /bin/bash
```